src/composables/useBlocknative.ts

```diff
@@ -1,12 +1,23 @@
+import { computed } from 'vue';
```

Comment:
```suggestion
import { computed, injected } from 'vue';
```

Can merge imports here